aea/test_tools/test_skill.py

```diff
@@ -67,6 +67,12 @@ def get_message_from_outbox(self) -> Optional[Message]:
         envelope = self._multiplexer.out_queue.get_nowait()
         return envelope.message
 
+    def dismiss_messages_from_outbox(self, number: int = 1) -> None:
```

Comment:
`drop` rather than `dismiss` is probably more explicit.

Reply:
Done